### DISCUSSION TITLE
Amiga HD disk image issue

### DISCUSSION THREAD

**Discussion by FlameMagister** [keirf/flashfloppy#572] [Q&A]
Hi all,

First time posting here and  thanks for all the posts and help guides here for flashfloppy - they really have been very useful and made installation very straightforward for me. 

That said, I am having an issue with high density disk images for the Amiga. I have set interface = amiga in the ff.cfg but I still  can't seem to read HD images on the Amiga. I keep getting block read errors. 

I have been creating blank HD images in WinUAE (using the 3.5" HD setting) and the gotek is one of the newer Artery models running 3.29 FF firmware. The Amiga is an A1200 model running OS 3.2. 

Sorry for this has been covered elsewhere or the answer is obvious and I grateful for any assistance. 

Kind regards, FM.

**Comment by AXm77** [comment]
Flash Gotek to older 3.13a firmware, I have same problem with OS 3.2, A3000 and FF 3.30 - error with HD ADFs.

**Reply by keirf** [reply]
It would be interesting to know which 3.x release introduced the regression.
